packages/dart/sshnoports/bundles/universal.sh

```diff
@@ -741,89 +758,21 @@ client() {
   # check that there are some ssh keys
   check_ssh_keys
 
-  # install the magic sshnp script
-  magic_script="$bin_path"/np.sh
-  cp "$extract_path"/sshnp/magic/sshnp.sh "$magic_script"
-  chmod +x "$magic_script"
-  if is_root && [ -f "$bin_path"/np.sh ]; then
-    ln -sf "$bin_path"/np.sh "$user_bin_dir"/np.sh
-    if [ $verbose = true ]; then
-      echo "=> Linked $user_bin_dir/np.sh to $bin_path/np.sh"
-    fi
-  fi
-
-  # get the inputs for the magic script
```

Comment:
No more np.sh

Reply:
np.sh was the "quick picker" script for sshnp sessions. But with both openssh and putty integrations documented, this script only offers the negative impact of a more challenging install experience.